src/eos-autoupdater.c

```diff
@@ -59,7 +59,7 @@ typedef enum _UpdateStep {
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
+static const char *UPDATE_STAMP_DIR = SHAREDSTATEDIR "/eos-updater";
```

Comment:
Oh, I guess you didn't override it. I can't see how sharedstatedir is set to /var/lib anywhere in autoconf or debhelper.